tests/circuit_graph/test_circuit_graph.py

```diff
@@ -61,24 +61,31 @@ def circuit(ops):
 
 
 @pytest.fixture
-def parameterized_circuit():
+def wires():
+    """A fixture for a mixed-type wires label list of length 3."""
+    return ['a', 'q1', 3]
+
+
```

Comment:
This runs the test on the "interesting" wires...

Reply:
Does this need to be a fixture? It seems more like test data

Reply:
You are right!

Reply:
But if it is test data, can I feed it into the fixture ``parametrized_circuit(wires)``? I am currently rather confused about this...

Reply:
yep, the fixture can use global/module variables in its definition